test/JDBC/expected/parallel_query/with_recompile-vu-verify.out

```diff
@@ -0,0 +1,3450 @@
+SELECT set_config('babelfishpg_tsql.explain_costs', 'off', false)
+go
+~~START~~
+text
+off
+~~END~~
+
+
+
+
+
+/*
+ * These tests validate the T-SQL RECOMPILE behaviour for stored procedures.
+ * The approach is to demonstrate the recompilation by disabling index scans, 
+ * and observing a different plan being generated only when RECOMPILE is used.
+ * Note that when a cached plan is used, there is no good way to prove in these tests that 
+ * it was indeed a cached plan, as it could just as well be a newly generated plan that 
+ * happens to be the same as the previous plan. While developing this fix, the PG planner() 
+ * function was instrumented to prove that cached plans are indeed used when plan_cache_mode={auto|force_generic_plan},
+ * but that is outside the scope of what can be used in these automated tests.
+ * This fix for forcing plans for statements in the procedure body to be recompiled only
+ * applies to paramtrized statements. Non-parametrized stmts, like the 2nd select in tb_recomp_11, 
+ * will not be recompiled since choose_custom_plan() will always return false for those.
+ * The current fix can be extended to also force such stmts to be recompiled but that is 
+ * left for later.
+ 
+ * Not tested:
+ * - recursive procedure calls, since Babelfish does not show plans when a conditional statement 
+ *   is involved (manual testing has shown RECOMPILE works as expected, i.e. the recursive call is
+ *   handled independently from the higher-level call)
+ * - plans inside SQL functions or triggers (not shown by PG when called from a procedure)
```

Comment:
A recursive call must always contain some condition that stops the recursion, otherwise it will recurse forever. Whenever a condition is present, Babelfish will not show the query plan.